src/entities/abTrail/ui/AbCanvas.tsx

```diff
@@ -287,6 +287,14 @@ const AbCanvas: FC<Props> = props => {
     });
   };
 
+  const onResult = () => {
```

Comment:
I think, there has been an issue with the naming of AbCanvas' `onLogResult` function. It should have been named `onResult` in the first place.
Regarding this, onResult function, I think it could stay as-is. But in order to avoid naming confusion I suggest calling  `props.onResult(...)` inside this function.